LICENSE

```diff
@@ -79,7 +79,7 @@ modification, are permitted provided that the following conditions are met:
   b. Redistributions in binary form must reproduce the above copyright
      notice, this list of conditions and the following disclaimer in the
      documentation and/or other materials provided with the distribution.
-  c. Neither the name of multipledispatch nor the names of its contributors
+  c. Neither the name of multiple dispatches nor the names of its contributors
```

Comment:
this is the name of the project and it shouldn't be changed

Reply:
got it thanks for suggestion @smichr